src/components/run-button-container.js

```diff
@@ -415,13 +407,6 @@ export function RunButtonContainer({ studyUuid, currentNode, disabled }) {
         studyUuid,
         limitReductionParam,
         currentNode?.id,
-        setRanLoadflow,
-        setShowContingencyListSelector,
-        setRanSA,
-        setRanSensi,
-        setRanShortCircuit,
-        setShowDynamicSimulationParametersSelector,
-        setRanDynamicSimulation,
```

Comment:
because :
https://legacy.reactjs.org/docs/hooks-reference.html#usestate
>  Note
>     React guarantees that setState function identity is stable and won’t change on re-renders. This is why it’s safe to omit from the useEffect or useCallback dependency list.